IdentityCore/src/webview/operations/MSIDWebResponseBrokerInstallOperation.m

```diff
@@ -1,116 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// All rights reserved.
-//
-// This code is licensed under the MIT License.
-//
-// Permission is hereby granted, free of charge, to any person obtaining a copy
-// of this software and associated documentation files(the "Software"), to deal
-// in the Software without restriction, including without limitation the rights
-// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
-// copies of the Software, and to permit persons to whom the Software is
-// furnished to do so, subject to the following conditions :
-//
-// The above copyright notice and this permission notice shall be included in
-// all copies or substantial portions of the Software.
-//
-// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-// THE SOFTWARE.
-
-
-#import "MSIDWebResponseBrokerInstallOperation.h"
-#import "MSIDWebviewResponse.h"
-#if TARGET_OS_IPHONE
-#import "MSIDBrokerInteractiveController.h"
-#endif
-#import "MSIDInteractiveTokenRequestParameters.h"
-#import "MSIDWebWPJResponse.h"
-#import "MSIDTokenRequestProviding.h"
-#import "MSIDKeychainTokenCache.h"
-#import "MSIDLegacyTokenCacheAccessor.h"
-#import "MSIDDefaultTokenCacheAccessor.h"
-#import "MSIDAADV2Oauth2Factory.h"
-#import "MSIDDefaultTokenResponseValidator.h"
-#import "MSIDAccountMetadataCacheAccessor.h"
-
-@interface MSIDWebResponseBrokerInstallOperation()
-
-@property (nonatomic) NSURL *appInstallLink;
-
-@end
-
-@implementation MSIDWebResponseBrokerInstallOperation
```

Comment:
Why did this file get removed?

Reply:
It was a "dead" code that was never used (confirmed with Kai and Antonio)